services/brig/test/integration/Main.hs

```diff
@@ -89,18 +90,25 @@ runTests iConf bConf otherArgs = do
     parseAWSEnv Nothing  = do
         sqsEnd   <- optOrEnv (Opts.sqsEndpoint . Opts.aws)      bConf parseEndpoint "AWS_SQS_ENDPOINT"
         dynEnd   <- optOrEnv (Opts.dynamoDBEndpoint . Opts.aws) bConf parseEndpoint "AWS_DYNAMODB_ENDPOINT"
-        sesEnd   <- optOrEnv (Opts.sesEndpoint . Opts.aws)      bConf parseEndpoint "AWS_SES_ENDPOINT"
-        sesQueue <- optOrEnv (Opts.sesQueue . Opts.aws)         bConf pack          "AWS_USER_SES_QUEUE"
         sqsIntQ  <- optOrEnv (Opts.internalQueue . Opts.aws)    bConf pack          "AWS_USER_INTERNAL_QUEUE"
         sqsJrnlQ <- join <$> optOrEnvSafe (Opts.userJournalQueue . Opts.aws) bConf (Just . pack) "AWS_USER_JOURNAL_QUEUE"
         dynBlTbl <- optOrEnv (Opts.blacklistTable . Opts.aws)   bConf pack          "AWS_USER_BLACKLIST_TABLE"
         dynPkTbl <- optOrEnv (Opts.prekeyTable . Opts.aws)      bConf pack          "AWS_USER_PREKEYS_TABLE"
-        return $ Opts.AWSOpts sesQueue sqsIntQ sqsJrnlQ dynBlTbl dynPkTbl sesEnd sqsEnd dynEnd
+        return $ Opts.AWSOpts sqsIntQ sqsJrnlQ dynBlTbl dynPkTbl sqsEnd dynEnd
 
     parseEndpoint :: String -> AWSEndpoint
     parseEndpoint e = fromMaybe (error ("Not a valid AWS endpoint: " ++ show e))
                     $ fromByteString (BS.pack e)
 
+    parseEmailAWSOpts :: IO (Maybe Opts.EmailAWSOpts)
+    parseEmailAWSOpts = case Opts.email . Opts.emailSMS <$> bConf of
+        Just (Opts.EmailAWS aws) -> return (Just aws)
+        Just (Opts.EmailSMTP  _) -> return Nothing
+        _                        -> do
+            sesEnd   <- parseEndpoint <$> getEnv "AWS_SES_ENDPOINT"
+            sesQueue <- pack          <$> getEnv "AWS_USER_SES_QUEUE"
+            return . Just $ Opts.EmailAWSOpts sesQueue sesEnd
```

Comment:
I'm not sure we need to keep supporting environment variables in tests; we're not using this anywhere. (removing environment variable support could be done later, too)

Reply:
> I'm not sure we need to keep supporting environment variables in tests; we're not using this anywhere. (removing environment variable support could be done later, too)

Good point. I actually added this "after" just for completeness.

> I'm not sure we need to keep supporting environment variables in tests; we're not using this anywhere. (removing environment variable support could be done later, too)

Let's do that then, totally agree to remove env variable usage at least in the tests.